UnitTests/Extensions/Test_BitmapIconExtensionMarkupExtension.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using Microsoft.Toolkit.Uwp.UI.Extensions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.AppContainer;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Markup;
+
+namespace UnitTests.Extensions
+{
+    [TestClass]
+    public class Test_BitmapIconExtensionMarkupExtension
+    {
+        [TestCategory("BitmapIconExtensionMarkupExtension")]
+        [UITestMethod]
+        public void Test_BitmapIconExtension_MarkupExtension_ProvideImage()
+        {
+            var treeroot = XamlReader.Load(@"<Page
+    xmlns=""http://schemas.microsoft.com/winfx/2006/xaml/presentation""
+    xmlns:x=""http://schemas.microsoft.com/winfx/2006/xaml""
+    xmlns:ex=""using:Microsoft.Toolkit.Uwp.UI.Extensions"">
+        <Button x:Name=""RootButton"">
+            <Button.Flyout>
+                <MenuFlyout>
+                    <MenuFlyoutItem Icon=""{ex:BitmapIcon Source=/Assets/StoreLogo.png}"" />
```

Comment:
@Sergio0694 if you use the full classname here, does that work any better in release mode? I wonder if there's an issue going on with the name mapping for some reason?

Reply:
Also, can you test if the samples work in Release mode as well with this similar sort of issue or not?

Reply:
As discussed, fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3110/commits/db20516409a3d507f75a6cf63da488902848dd88 and https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3110/commits/364ea74a6ec2ed3425ed419cc246b520b74b6a81 🎉